WordPress/src/test/java/org/wordpress/android/viewmodel/pages/PagesViewModelTest.kt

```diff
@@ -183,6 +188,26 @@ class PagesViewModelTest {
         assertThat(viewModel.arePageActionsEnabled).isTrue()
     }
 
+    @Test
+    fun `Auto-upload is initiated when the user enters the screen`() {
+        // Given
+        // When
+        viewModel.start(site)
+        // Then
+        verify(uploadStarter).queueUploadFromSite(site)
+    }
+
+    @Test
+    fun `Auto-upload is initiated when the user pulls to refresh`() {
+        // Given
```

Comment:
Same as above 👍

Reply:
Fixed in 143453475e547212b89d1991f7ecf1cca98e2bec